addons/web/static/src/js/views/kanban/kanban_model.js

```diff
@@ -293,6 +295,34 @@ var KanbanModel = BasicModel.extend({
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * @override
+     */
+    _getGroupedListPropsToKeep(list, options) {
+        const propsToKeep = this._super(...arguments);
+        if (list.domainExtension) {
+            propsToKeep.domainExtension = list.domainExtension;
+            if (options.onlyGroup) {
+                // when we have a domain extension - such as an active
+                // progressbar filter - and we will not search_read the
+                // group, then do not keep count and res_ids props.
+                // This is important for the case where we activate a
+                // progressbar filter for an "other" progressbar field value.
+                // Keeping those props in this case will badly compute
+                // the progressbar counters for this "other" field value.
+                delete propsToKeep.res_ids;
+                delete propsToKeep.count;
+            }
```

Comment:
need to check if its still useful

Reply:
It was not since read_group rpcs that led to this bad behavior are now replaced by search_read rpcs or abandoned.